Library/Homebrew/cmd/deps.rb

```diff
@@ -64,51 +65,68 @@ def deps
     Formulary.enable_factory_cache!
 
     recursive = !args.send("1?")
-    installed = args.installed? || args.formulae.all?(&:opt_or_installed_prefix_keg)
```

Comment:
Was the intention of `opt_or_installed_prefix_keg` here just to check if a version was installed, or is there some subtlety that I'm missing?

Reply:
Yeh, I think so. Could instead use `.any_version_installed?` I think here.